### PR TITLE
Feature EML-596: Fixed documentation with new imports

### PR DESCRIPTION
**Pull request by valenvivaldi**

### CODE REVIEW

File: docs/developer-guide/etendo-copilot/how-to-create-copilot-tools.md [hunk 1]
```diff
@@ -58,41 +58,28 @@ For this example, we will create a tool that will allow us to make a ping to a h
...
-        def run(self, host: str, *args, **kwargs): # The run method is the one that will be executed when the tool is executed. The inputs of the method are the inputs of the tool.
-            import requests # Import the necessary libraries to execute the tool. It is recommended to import the libraries inside the run method to avoid conflicts with other tools.
-            response = requests.get(host)
-            return {"status_code": response.status_code}  # The run method must return a dictionary with the outputs of the tool.
-    ``` 
-    !!!Note 
-        The name of the tool must be the same as the name of the class that extends the ToolWrapper class.
+    
 
 3. In case the tool needs more than one input, its necessary to use a Dict as input. In order to do that, we have to create a new class that defines the inputs of the tool using pydantic. Here is an example of a tool that receives a Dict as input, with the structure of the Dict defined in a class:
```
Comment (isaiasb-etendo) [inline docs/developer-guide/etendo-copilot/how-to-create-copilot-tools.md:63]: Change the numbers, as you have deleted an item

Comment (isaiasb-etendo) [inline docs/developer-guide/etendo-copilot/how-to-create-copilot-tools.md:63]: Now a dict is always used, isn't it?

File: docs/developer-guide/etendo-copilot/how-to-create-copilot-tools.md [hunk 2]
```diff
@@ -58,41 +58,28 @@ For this example, we will create a tool that will allow us to make a ping to a h
...
+    
+    !!!Warning 
+        The name of the tool must be the same as the name of the class that extends the ToolWrapper class.
```
Comment (isaiasb-etendo) [inline docs/developer-guide/etendo-copilot/how-to-create-copilot-tools.md:66]: Tool Search Key, is the one that has to match, the name can be different

image.

**Comment by valenvivaldi** [comment]
@isaiasb-etendo Ive updated the PR.

**Review by isaiasb-etendo — APPROVED** [review]

**Review by valeg-etendo — APPROVED** [review]

**Pull request merged**